### PR TITLE
Adds scripts to initialize Cassandra

### LINKED ISSUE #29: Model relations and tables to microservices, nodes and services
We need to create the model and relations in Cassandra to our model.

Basically, our approach is the next:

* We have unlimited microservices to monitor.
* Each microservice has to have unless one node.
* Each node will have services to monitoring. 
* These services will export the metrics and values.

### LINKED ISSUE #30: Script to create the database, tables and inserts in Cassandra as initial content.
After #29 we need to create a script to create these tables and relations in Cassandra.

### PR DESCRIPTION
**Pull request by eperinan**
It resolves #29 and resolves #30 

Thanks for your advices @FPerezP. Finally I added as ordering keys the dates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,20 @@ Opscenter
 
 The previous command will overwrite this proto file.
 
+## Requirements
+
+As database, we need install Cassandra. We use this database to store the microservices, nodes and services which we are monitoring.
+These tables will be updated directly from each microservices. 
+
+Run this command to initialize the database.
```
Comment (FPerezP) [inline README.md:16]: We should create the schema at startup time if the Keyspace or tables don't exist, in the same way Akka Persistence works.

File: scripts/cassandra-init.cql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// Keep in mind that we have to change the class replication to NetworkTopologyStrategy in production.
+CREATE KEYSPACE IF NOT EXISTS freestyle_opscenter WITH replication = {'class': 'SimpleStrategy', 'replication_factor' : 1};
+
+DROP TABLE IF EXISTS freestyle_opscenter.microservices;
```
Comment (FPerezP) [inline scripts/cassandra-init.cql]: Why do we need to drop the table?

**Comment by FPerezP** [comment]
Great job @eperinan!! Just a couple of comments. 
Creating the schema at startup time if it doesn't exist can be done in a separate ticket if it makes mroe sense to you. 
Other than that, looks good!

**Comment by eperinan** [comment]
Makes sense @FPerezP. Totally agree with you. Feel free to create the new ticket for this and address it if you want :-p

**Comment by eperinan** [comment]
Make sense @FPerezP. I really appreciate your review. I am going to create the issue for your proposal.

**Review by FPerezP — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (eperinan) [reply README.md:16]: Makes sense. Ticket for this here https://github.com/frees-io/freestyle-opscenter/issues/36

**Review by FPerezP — APPROVED** [review]

**Pull request merged**